README.rst

```diff
@@ -125,7 +125,7 @@ This creates :code:`PS1.npy`, which contains the full power spectrum. The next s
 ::
 
   $ sagpr_get_PS -lm 0 -f coords_1000.xyz -o PS0
-  $ sagpr_get_kernel -z 2 -ps PS1.npy -ps0 PS0.npy -s PS0_natoms.npy -o kernel1
+  $ sagpr_get_kernel -z 2 -ps PS1.npy -ps0 PS0.npy -s PS1_natoms.npy -o kernel1
```

Comment:
For this one, it doesn't matter which one you use since they're identical (they're just the numbers of atoms). But, best practice is to do what you've changed it to